tests/helpers/test_selector.py

```diff
@@ -373,7 +402,6 @@ def test_select_selector_schema(schema, valid_selections, invalid_selections):
     (
         {},  # Must have options
         {"options": {"hello": "World"}},  # Options must be a list
-        {"options": []},  # Must have at least option
```

Comment:
We should still have this check? If custom_value is `False`, we still would need at least one?